Test/Tests/Network/FloatingIP.Tests.ps1

```diff
@@ -89,9 +92,22 @@ Describe "Floating IP" {
 
                 $MultiNode.NM.AddPolicyToNetwork($ContrailPolicy, $ContrailClientNetwork)
                 $MultiNode.NM.AddPolicyToNetwork($ContrailPolicy, $ContrailServerNetwork)
+
+                foreach ($Session in $MultiNode.Sessions) {
+                    Initialize-DockerNetworks `
+                    -Session $Session `
+                    -Networks $Networks `
+                    -Configs $MultiNode.Configs `
+                }
+
             }
 
             AfterAll {
+                foreach ($Session in $MultiNode.Sessions) {
+                    Remove-DockerNetwork -Session $Session -Name $ServerNetwork.Name
```

Comment:
Since this PR introduces `$Networks`, we can just iterate over it instead of calling it for each network separately.